hvac/v1/__init__.py

```diff
@@ -1623,7 +1623,7 @@ def get_kubernetes_configuration(self, mount_point='kubernetes'):
         return self._adapter.get(url)
 
     def create_kubernetes_role(self, name, bound_service_account_names, bound_service_account_namespaces, ttl="",
-                               max_ttl="", period="", policies=None, mount_point='kubernetes'):
+                               max_ttl="", period="", policies=None, mount_point='kubernetes', token_type=""):
```

Comment:
Mind moving this new kwarg to before `mount_point`? We typically try to keep that argument at the end of a method's signature for consistency reasons:
```suggestion
                               max_ttl="", period="", policies=None, token_type="", mount_point='kubernetes'):
```